### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuliiaSaliuta**

### CODE REVIEW

**Review by SevKrok — APPROVED** [review]
Nice 👍

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+}
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    result = requests.get(URL, params=PARAMS)
+    json_result = json.loads(result.content)
+
+    print(json_result)
```
Comment (Y-Havryliv) [inline app/main.py]: do you need this print?

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+
+    date = json_result["location"]["localtime"]
+    temperature = json_result["current"]["temp_c"]
+    condition = json_result["current"]["condition"]["text"]
+
+    print(
+        f"Paris/France {date} Weather: {temperature} Celsius, {condition}"
```
Comment (Y-Havryliv) [inline app/main.py]: why did you hardcode it? It's better to take city and country from parsed result

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,32 @@
+import os
+import json
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
```
Comment (LiudmylaKulinchenko) [inline app/main.py:7]: as for me, better to name it city, not filtering

**Review by LiudmylaKulinchenko — APPROVED** [review]
Nice work! Also added you some recomendation 👇

File: app/main.py (hunk 3 above)
Comment (LiudmylaKulinchenko) [reply app/main.py:7]: I got it, you have an example in checklist with such naming. It's totally fine 🙌

**Pull request closed without merging**